frameworks/sveltekit.md

```diff
@@ -117,6 +117,24 @@ const config = {
 export default config
 ```
 
+In addition to the configuration above, it's necessary to explain SvelteKit on how to handle and insert the webmanifest into the generated source code. Currently the easiest way to do this, is to add a layout to your kit project.
```

Comment:
weird, replace with:

In addition to the configuration above, it's necessary to add the PWA webmanifest., currently the easiest way to do this, is to add a layout to your kit project.